.gitlab-ci.yml

```diff
@@ -96,6 +96,22 @@ build:
         buildah push --format=v2s2 "$CONTAINER_REPO:$DOCKER_IMAGE_TAG"
       fi
 
+build-bench:
+  stage: build
+  before_script:                   []
+  variables:
+    POLKADOT_ADDRESS:              "ws://polkadot-node.nodes:9944"
+  image:                           "alvicsam/sidecar-bench:3.0"
+  script:
+    - export SAS_SUBSTRATE_WS_URL=$POLKADOT_ADDRESS
+    - yarn
+    - yarn build
+    - yarn run main &
+    - sleep 10
+    - wrk -d10m -t4 -c12 --timeout 120s --latency -s ./ci/benchmarks/lightweight-bench.lua "http://127.0.0.1:8080"
```

Comment:
This all seems incredibly flexible, thanks so much again! So if I understand this correctly we can actively update our benchmarking scripts through this directory, and fix any connecting calls through this yml, and it should proactively run the updated code?

Reply:
Yes, that's correct. I'm still experimenting and it seems that 10 minutes shows more accurate result.
This job runs on each commit. If we merge it with current settings it'll run on each commit in each pr and on master/tag as well.